server/lib/games.ts

```diff
@@ -33,7 +33,8 @@ export function createGame(
     drawPileCards: [],
     discardPileCards: [],
   };
-  console.log(JSON.stringify(games, null, 4));
+  //commented out for now
+  // console.log(JSON.stringify(games, null, 4));
```

Comment:
comment for a comment :) I would delete this. Btw without stringify you get much nicer output.

Reply:
I know this seems stupid, it's just for me so I don't lose track of why commented out certain lines 😅